client/src/views/Feed/Feed.vue

```diff
@@ -65,31 +66,49 @@ export default {
     }
   },
   created () {
-    let route = ''
-    if (Authorized.isAuthorized()) {
-      route = '/api/feed/authorized'
-    } else {
-      route = '/api/feed/anonimus'
+    this.loadFeedData(this.$route.query.tag)
+  },
+  watch: {
```

Comment:
Why you watch route instead of using route hooks -https://router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards ?